flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/rank/window/processors/RowTimeWindowRankProcessor.java

```diff
@@ -140,6 +140,11 @@ public void open(Context<Long> context) throws Exception {
         this.currentProgress = Long.MIN_VALUE;
     }
 
+    @Override
+    public void initializeWatermark(long watermark) {
```

Comment:
how about just use initializeProgress? keep aligned with advanceProgress

Reply:
I prefer initializeWatermark.
For event time, progress is equals to watermark.
But for proctime, progress is processtime time.
Here we only initialize watermark.